### ISSUE TITLE
GC: search for xDAI does not return xDAI token in the list

### ISSUE THREAD

**Issue by anxolin** [cowprotocol/cowswap#330]
1. Connect to a wallet inGC
2. Open a token list
3. Type XDAI into the field

**AR**: No xDAI in the list
![image.png](https://images.zenhubusercontent.com/5f55ec970a2e890ec10f7778/c66c7c57-c467-4577-b432-682e0bcf944f)

**ER**: xDAI shoule be the 1st in the list

> Original issue https://github.com/gnosis/cowswap/issues/2613 by @elena-zh

**Comment by alfetopito** [comment]
Is this an old issue?

I can see it fine using the release branch

![Screen Shot 2022-05-03 at 11 38 45](https://user-images.githubusercontent.com/43217/166441290-d1da5c69-1648-406f-922c-8c2d7e64ca4f.png)
![Screen Shot 2022-05-03 at 11 38 37](https://user-images.githubusercontent.com/43217/166441292-d8c91a2d-874a-43fe-a1e9-e9848354430b.png)

**Comment by elena-zh** [comment]
Then, it might be fixed there. Due to the issues is still reproducible in Prod. 

Btw, I have found a new issue: incorrect message appears when 'select token' is  displayed in the from field 
![image](https://user-images.githubusercontent.com/70885163/166442501-983d53ab-0f2b-4f2b-a006-2e56e1d55589.png)

Here we should show 'Enter an amount' 
![image](https://user-images.githubusercontent.com/70885163/166442613-fcb06ddb-a3c9-4cea-97da-f274473d8321.png)

OR 'Select a token' messages
![image](https://user-images.githubusercontent.com/70885163/166442659-af7afa47-1cc6-4cff-814d-f8582e07da93.png)

I have created #504 issue for this.
